packages/bot-skeleton/src/scratch/hooks/contextmenu.js

```diff
@@ -177,3 +177,36 @@ Blockly.ContextMenu.blockCommentOption = function (block) {
 
     return comment_option;
 };
+
+/**
+ * Make a context menu option for detaching the current block.
+ * deriv-bot: Use Blockly's implementation.
+ * @param {!Blockly.BlockSvg} block The block where the right-click originated.
+ * @return {!Object} A menu option, containing text, enabled, and a callback.
+ * @package
+ */
+Blockly.ContextMenu.blockDetachOption = function (block) {
+    const canDisconnect = block
```

Comment:
Just a reminder for us to think about on Monday. `canDisconnect` is ok mostly, but let's consider some options:
`isDisconnectable`
`shouldDisconnect`
`ableToDisconnect`